src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -121,11 +121,11 @@
 {% block file_widget -%}
     <{{ element|default('div') }} class="custom-file">
         {%- set type = type|default('file') -%}
-        {{- block('form_widget_simple') -}}
-        {%- set label_attr = label_attr|merge({ class: (label_attr.class|default('') ~ ' custom-file-label')|trim }) -%}
         {%- set input_lang = 'en' -%}
         {% if app is defined and app.request is defined %}{%- set input_lang = app.request.locale -%}{%- endif -%}
-        <label for="{{ form.vars.id }}" lang="{{ input_lang }}" {% with { attr: label_attr } %}{{ block('attributes') }}{% endwith %}>
+        {% with { attr: {lang: input_lang} | merge(attr) } %}{{ block('form_widget_simple') }}{% endwith %}
```

Comment:
no need to use a `with` block here. you could set the value in the `attr` variable directly.

Reply:
you're right. I applied this, makes it also more readable :)